### ISSUE TITLE
PFHB02 Issue

### ISSUE THREAD

**Issue by StellarExile** [KOTORCommunityPatches/K1_Community_Patch#433]
Apparently there's a weird issue with PFHB02 that causes the head to have multiple/distorted eyes when displaying its dark side transitions:
![PFHB02 Eyes](https://user-images.githubusercontent.com/61084144/93029866-74453100-f5ec-11ea-9e39-c89a0fecbf6b.jpg)
Not sure if it's a UV or tga problem.

**Comment by DarthParametric** [comment]
I remember Dark Hope bringing it up on DS once, and [I posted a fix for it](https://www.darthparametric.com/files/kotor/k1/[K1]_PFHB02_DS_Transition_Eye_Fix.7z), although I can't seem to find the original post. Looks like it must have been back in May, judging by the dates on the archive.

From memory I just took the LS eye and did a graduated yellowing of it, but I'm not sure how well it fits the vanilla style. If you are willing, take a look and make an assessment.
